include/afl-framework.h

```diff
@@ -0,0 +1,19 @@
+#include "afl-fuzz.h"
+#include "list.h"
+
+typedef struct afl_executor
+{
+    bool has_forkserver;
+    bool has_fauxserver;
+    afl_state_t * afl_state;
+    u32 map_size;
+} afl_executor_t;
```

Comment:
would there not be a forkserver state? or is that in the afl_state?
and why is the map_size extra? would that not be in the afl_state or the forkserver_state? just asking so I understand this

Reply:
The forkserver state is in the afl_state itself. Regarding the map_size, we set the map size through an environment variable or it is the default value, Since we're not exactly exposing the real forkserver struct (which finally contains the map size) I thought including it here would be a good way.

Reply:
then maybe "initial_map_size" or "map_size_default" or something like that?
if it is needed?